history/characters/6000_gurubashi.txt

```diff
@@ -373,7 +374,39 @@
 	580.8.23={ birth=yes add_trait=creature_troll }
 	654.7.12={ death=yes }
 }
-
+6039={
+	name=Drukinjo
+	dynasty=8097
+	culture=gurubashi religion=cult_of_hakkar
+	martial=9 diplomacy=8 stewardship=7 intrigue=7 learning=5
+	481.12.16={ birth=yes add_trait=creature_troll }
+	554.7.12={ death=yes }
+	487.12.16={
+		add_trait=class_loa_priest_6
+		add_trait=stubborn
+		add_trait=honest
+		add_trait=diligent
+		add_trait=lustful
+		add_trait=ambitious
+		add_trait=zealous
+		add_trait=outstanding_erudite
+}
+6040={
+	name=Sinthal
+	dynasty=8414
+	culture=gurubashi religion=cult_of_hakkar
+	martial=9 diplomacy=4 stewardship=6 intrigue=8 learning=3
```

Comment:
Too OP, particularly in Martial/Intrigue. Try balancing his stats around the total number of 24.